packages/manager/modules/error-page/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "overrides": [
+    {
+      "files": ["**/*.md"],
+      "rules": {
+        "import/no-extraneous-dependencies": "off"
+      }
+    }
```

Comment:
I notice this is added for almost every app. Does it make sense to add it to the global eslint config ? Or do we want to get rid of it app by app ?

Reply:
You're right! It will be easier to update if needed so that's why I moved all overrides for `*.md` files into the root ESLint configuration.